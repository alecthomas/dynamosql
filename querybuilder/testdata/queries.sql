SELECT * FROM gamescores WHERE UserId = :UserId
SELECT * FROM gamescores WHERE UserId = :UserId AND TopScore > :MinTopScore
SELECT * FROM gamescores WHERE UserId = :UserId AND GameTitle BETWEEN :MinGameTitle AND :MaxGameTitle AND TopScore > :MinTopScore
SELECT * FROM gamescores WHERE UserId = "103" AND GameTitle BETWEEN "Galaxy" AND "Meteor" AND TopScore > 1000
