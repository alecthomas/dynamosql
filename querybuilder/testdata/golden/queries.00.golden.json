{
  "Query": "SELECT * FROM gamescores WHERE UserId = :UserId",
  "Prepared": {
    "Query": {
      "AttributesToGet": null,
      "ConditionalOperator": null,
      "ConsistentRead": null,
      "ExclusiveStartKey": null,
      "ExpressionAttributeNames": null,
      "ExpressionAttributeValues": null,
      "FilterExpression": null,
      "IndexName": null,
      "KeyConditionExpression": "UserId = :UserId",
      "KeyConditions": null,
      "Limit": null,
      "ProjectionExpression": null,
      "QueryFilter": null,
      "ReturnConsumedCapacity": null,
      "ScanIndexForward": null,
      "Select": null,
      "TableName": "gamescores"
    },
    "FreeParams": {
      ":UserId": {}
    },
    "FixedParams": {}
  }
}
